tests/integ/modin/groupby/test_groupby___iter__.py

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+import modin.pandas as pd
+import pandas as native_pd
+import pytest
+
+import snowflake.snowpark.modin.plugin  # noqa: F401
+from tests.integ.modin.utils import assert_frame_equal, assert_series_equal
+from tests.integ.utils.sql_counter import SqlCounter, sql_count_checker
+
+
+@pytest.mark.parametrize(
+    "by, expected_query_count, expected_join_count",
+    [
+        ("a", 3, 4),
+        (["a"], 3, 4),
+        (["a", "b"], 4, 6),
```

Comment:
Maybe the case when `by= (None,)` could be added here  since`_return_tuple_when_iterating=False` then?

Reply:
`since_return_tuple_when_iterating=False` is already tested using `by="a"`. The other two test cases result in `since_return_tuple_when_iterating=True`.

Reply:
I see, in that case it's not needed.